src/serac/physics/heat_transfer.hpp

```diff
@@ -444,7 +444,8 @@ class HeatTransfer<order, dim, Parameters<parameter_space...>, std::integer_sequ
   {
     residual_->AddDomainIntegral(
         Dimension<dim>{}, DependsOn<0, 1, 2, active_parameters + NUM_STATE_VARS...>{},
-        [source_function, this](auto x, auto temperature, auto /* dtemp_dt */, auto shape, auto... params) {
+        [source_function, this](double /*t*/, auto x, auto temperature, auto /* dtemp_dt */, auto shape,
```

Comment:
We should not capture `this`. `ode_time_point_` is used incorrectly on line 462 as a result.